Source/Core/VideoBackends/OGL/Render.cpp

```diff
@@ -1459,6 +1460,15 @@ void Renderer::SwapImpl(u32 xfbAddr, u32 fbWidth, u32 fbStride, u32 fbHeight,
   UpdateActiveConfig();
   g_texture_cache->OnConfigChanged(g_ActiveConfig);
 
+  // Invalidate shader cache when the host config changes.
+  u32 new_host_config_bits = g_ActiveConfig.GetHostConfigBits();
```

Comment:
HostConfigBits instead of u32, and I'd like to see this in VideoCommon. Maybe with a OnConfigChange(oldconfig, newconfig) method.